Reference/Config/umbracoSettings/index.md

```diff
@@ -474,212 +327,104 @@ Let's have a further look at each option below.
-
-For each task you want to run you should add a **`<task>`** element.
-
-The task elements consist of the following attributes:
-
-**log:** Set this to true if you want to write to the umbracoLog table and see if everything is working as expected. If set to false nothing will be written to the log.
-
-**alias:** The alias is being used in the log so you can distinguish between the different tasks and other log-entries.
-
-**interval:** The interval is set in seconds and determines how often the task should be run.
-
-**url:** Here the url for the page that should be called to run the task must be entered. Please note this can also point to an extensionless url or a service etc.
-
-**Please note:** that the scheduler is not in any way a windows process so it depends on the application pool in which Umbraco is located. This means that if the application pool resets, so will the scheduler, so this is not a highly reliable way of scheduling tasks.
-
-## DistributedCalls / Loadbalancing
-
-**Please note:** this setting applies only to [legacy load balancing](../../../Getting-Started/Setup/Server-Setup/Load-Balancing/traditional.md).
+### <maxLogAge>
 
-Umbraco comes with ability to distribute its cached content to multiple servers via a method known as load balancing. Umbraco has to be installed on all servers, with all servers sharing the same database.
+The maximum log age in minutes used for the internal audit log scrubbing.
 
-*Prior to version 7.3*, when the Umbraco instances are setup and files are synced between the instances, the instances need to know when to refresh their cache. This happens in the `<servers>` setting.
-
-Every time some content is published in Umbraco. You can ask Umbraco to ping other hosts and tell them to update their cache. Make sure that these instances can be reached internally on port 80.
-
-Also, remember to include the actual instance performing the publish if you it to refresh its own cache as well. Cache will only be refreshed on servers in the list.
-
-```xml
-<!-- distributed calls make Umbraco use webservices to handle cache refreshing -->
-<distributedCall enable="false">
-    <!-- the id of the user who's making the calls -->
-    <!-- needed for security, Umbraco will automatically look up correct login and passwords -->
-    <user>0</user>
-    <servers>
-        <!-- add ip number or hostname, make sure that it can be reached from all servers -->
-        <!-- you can also add optional attributes to force a protocol or port number (see #2) -->
-        <!-- <server>127.0.0.1</server>-->
-        <!-- <server forceProtocol="http|https" forcePortnumber="80|443">127.0.0.1</server>-->
-    </servers>
-</distributedCall>
-```
+## Web.Routing
 
-As all instances share the same database the only authentication we need between the servers are the ID of users making the call, which is set in the `<users>` element. This is strictly for logging.
+This section configures...
```

Comment:
This sentence needs to be completed - right?

Reply:
It's the same as the old one.  But it would be better to replace with something better :-)

Reply:
removed the line